index.js

```diff
@@ -3,7 +3,7 @@ const mkdirp = require('mkdirp');
 const spdlog = require('bindings')('spdlog');
 
 exports.version = spdlog.version;
-exports.setAsyncMode = spdlog.setAsyncMode;
+exports.initThreadPool = spdlog.initThreadPool;
```

Comment:
what is the use case for an app to control the global thread pool ? If the app creates multiple worker threads in the pool, the message ordering is not guaranteed in the implementation https://github.com/gabime/spdlog/wiki/6.-Asynchronous-logging#message-ordering. I think we should just stick to the default implementation.

Reply:
Single worker thread is indeed recommended. Also note that if there is only single worker then single threaded(_st) loggers can be used instead of _mt.